ls1intum/Artemis
# Fix assessing new models when retrieving them for assessment

### Motivation and Context
When a student submits a modeling submission after the remaining submissions assessed, Compass was not suggesting automatic assessments on submissions page of exercise. This feature was working on assessment dashboard but not submissions page.

### Steps for Testing
1. Add a submission
2. Manually assess it
3. Create a new submission
4. Go to exercise/submissions
5. Assess model
6. Suggestions must appear
